pkg/devfile/adapters/kubernetes/utils/utils.go

```diff
@@ -44,6 +44,21 @@ func GetContainers(devfileObj devfile.DevfileObj) []corev1.Container {
 	return containers
 }
 
+// GetVolumes iterates through the components in the devfile and returns a map of component alias to the devfile volumes
+func GetVolumes(devfileObj devfile.DevfileObj) map[string][]common.DockerimageVolume {
```

Comment:
Should the return type be something more generic than `common.DockerimageVolume`? When we support Kubernetes or other component types might be weird for them to have to convert to use the Docker image the type. I'm on the fence about this though because I think if we do create a generic type it will likely have the same properties.

Reply:
updated